backend/server/routers/programs.py

```diff
@@ -211,4 +212,21 @@ def get_structure(
             add_subgroup_container(structure, "General", container, [])
 
     
-    return {"structure": structure}
\ No newline at end of file
+    return {"structure": structure}
+
+@router.get(
+    "/getGenEds/{programCode}",
+    responses={
+        400: {
+            "description": "The given program code could not be found in the database",
+        },
+        200: {
+            "description": "Returns all geneds available to a given to the given code",
+        },
+    },
+)
+
+def getGenEds(
+    programCode: str, search = None):
+    all_geneds = data_helpers.read_data("data/scrapers/genedPureRaw.json")
+    return {"general education courses" : all_geneds[programCode]}
```

Comment:
Im assuming this is the key if the payload for this endpoint? Really nitpick but can we name this to 'courses' instead? (if that's better? Haha) 😅